### PR TITLE
Add missing native ProveEnterUsername component

### PR DESCRIPTION
**Pull request by chromakode**
I added a missing component to the native dumb sheet. Sorry about that!

:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by MarcoPolo** [comment]
👍

File: shared/profile/prove-enter-username.native.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// @flow
+import React, {Component} from 'react'
+import type {Props} from './prove-enter-username'
+
+class Render extends Component<void, Props, State> {
```
Comment (MarcoPolo) [inline shared/profile/prove-enter-username.native.js]: State isn't defined?

**Comment by MarcoPolo** [comment]
👍

File: shared/profile/prove-enter-username.native.js (hunk 1 above)
Comment (chromakode) [inline shared/profile/prove-enter-username.native.js]: :sweat:

**Pull request merged**